services/hdfs/README.md

```diff
@@ -1,85 +1,84 @@
+Before proceeding, make sure you have a Kubernetes cluster running and `kubectl` is configured to use the cluste and familiarize yourself with the following resoruce files:
+- [configmap.yaml](./configmap.yaml)
+- [datanodes.yaml](./datanodes.yaml)
+- [namenode.yaml](./namenode.yaml)
 
-```
-helm repo add stackable-stable https://repo.stackable.tech/repository/helm-stable/
-```
-And install the required Kubernetes operators for running an HDFS cluster.
-```
-helm install --wait zookeeper-operator stackable-stable/zookeeper-operator --version 23.7.0
-helm install --wait hdfs-operator stackable-stable/hdfs-operator --version 23.7.0
-helm install --wait commons-operator stackable-stable/commons-operator --version 23.7.0
-helm install --wait secret-operator stackable-stable/secret-operator --version 23.7.0
-```
+Once ready, apply the following commands to deploy the HDFS cluster in the following order
 
-Then we are ready to deploy HDFS and its required services.
+1. Deploy the configmap
+````bash 
+kubectl apply -f configmap.yaml
+````
 
-### Set up an HDFS cluster 
-The three yaml-files `zk.yaml`, `znode.yaml`, and `hdfs.yaml` have been taken directly from [Set up an HDFS cluster](https://docs.stackable.tech/home/stable/hdfs/getting_started/first_steps)
+2. Ensure it's deployed
 
+````bash
+kubectl get configmap hadoop-config
+````
 
-#### Setup Zookeeper
-Apply the two yaml-files: `zk.yaml` and `znode.yaml`.
-```
-kubectl apply -f zk.yaml
-kubectl apply -f znode.yaml
-```
-And wait for Zookeeper deployment to complete.
+3. Deploy the namenode.yaml
 
-```
-kubectl rollout status --watch --timeout=5m statefulset/simple-zk-server-default
-```
+````bash
+kubectl apply -f namenode.yaml
+````
 
+4. Ensure the pod is created successfully
 
-#### Setup HDFS
-Apply the single yaml-file: `hdfs.yaml`.
-```
-kubectl apply -f hdfs.yaml
-```
+````bash
+kubectl get pod -w
+kubectl describe pod namenode-<ID>
+kubectl logs namenode-<ID>
+````
 
-And wait for HDFS deployment to complete.
-```
-kubectl rollout status --watch --timeout=5m statefulset/simple-hdfs-datanode-default
-kubectl rollout status --watch --timeout=5m statefulset/simple-hdfs-namenode-default
-kubectl rollout status --watch --timeout=5m statefulset/simple-hdfs-journalnode-default
-```
+5. Deploy the datanodes.yaml
+
+````bash
+kubectl apply -f datanodes.yaml
+````
+
+6. Ensure that 3 datanode pods are created 
+
+````bash
+kubectl get pod -w 
+````
 
 ### Verify HDFS works
-Deploy a temporary helper pod `webhdfs.yaml` to interact with HDFS.
-```
-kubectl apply -f webhdfs.yaml
-```
-And wait for completion:
-```
-kubectl rollout status --watch --timeout=5m statefulset/webhdfs
-```
+Create a connection to namenode pod using port-forwarding as below:
 
-We expect the HDFS cluster to be empty once installed. The following alias will used for accessing the content of the root directory in HDFS by the helper pod. 
+```bash
+kubectl port-forward svc/namenode 9870:9870
```

Comment:
Should this not be the `kubectl port-forward svc/namenode 9000:9000` because 9000 is the HTTP post and 9870 is the RPC port?

```suggestion
kubectl port-forward svc/namenode 9000:9000
```

Reply:
No. Because I do this forwarding with respect to running a curl cmd on my localhost